app/controllers/course/controller.rb

```diff
@@ -38,6 +38,7 @@ def current_component_host
       Course::ControllerComponentHost.new(current_tenant.settings(:components),
                                           current_course.settings(:components), self)
   end
+  helper_method :current_component_host
```

Comment:
You can get the component host by `controller.current_component_host`, think you don't need to make this a helper.